probes/pom.xml

```diff
@@ -16,6 +16,11 @@
 
     <dependencies>
         <dependency>
+            <groupId>commons-codec</groupId>
```

Comment:
I guess the newly added JAR will have to be added to Provisioner::installAgent ?